README.md

```diff
@@ -148,6 +148,7 @@ set -g @catppuccin_window_current_background "#{@thm_mauve}"
 
 # Load catppuccin
 run ~/.config/tmux/plugins/catppuccin/tmux/catppuccin.tmux
+# For TPM, instead use `run ~/.config/tmux/plugins/tmux/catppuccin.tmux`
```

Comment:
This is the exact same line as above, right?

Reply:
You are correct, they are the same script, but the location differs. I was attempting to call out because if you are scanning the docs it's easy to miss. Especially, if you don't realize that TPM places the theme in a `tmux` directory instead of `catppuccin` and don't read the manual installation step earlier in the docs.

Reply:
Oh wow, you have no idea how long I stared at this. Thanks!